sections/03_categorize.md

```diff
@@ -236,7 +253,9 @@ making methodological choices that either reduce the need for labeled examples
 or that use transformations to training data to increase the number of times it
 can be used before overfitting occurs. For example, the unsupervised and
 semi-supervised methods that we've discussed reduce the need for labeled
-examples [@doi:10.1016/j.jbi.2016.10.007]. The adversarial training example
+examples [@doi:10.1016/j.jbi.2016.10.007]. The anchor and learn framework
+[@pmid:27107443] uses expert knowledge to identify high confidence
```

Comment:
Can you make this a @doi tag so we can automatically snag the reference information?
I think the DOI is: 10.1093/jamia/ocw011